addons/pos_pinelabs/models/pinelabs_pos_request.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import logging
+
+from odoo import _
+from odoo.addons.pos_pinelabs import const
+REQUEST_TIMEOUT = 10
+
+_logger = logging.getLogger(__name__)
+
+class PinelabsPosRequest:
+    def __init__(self, payment_method):
+        self.pinelabs_security_token = payment_method.pinelabs_security_token
+        self.pinelabs_merchant = payment_method.pinelabs_merchant
+        self.pinelabs_allowed_payment_modes = payment_method.pinelabs_allowed_payment_modes
+        self.pinelabs_store = payment_method.pinelabs_store
+        self.pinelabs_client = payment_method.pinelabs_client
+        self.payment_method = payment_method
+        self.session = requests.Session()
+
+    def _get_pinelabs_endpoint(self):
+        return 'https://www.plutuscloudserviceuat.in:8201/API/CloudBasedIntegration/V1/'
+
+    def _call_pinelabs(self, endpoint, payload):
+        """ Make a request to Pine Labs POS API.
+
+        :param str endpoint: The endpoint to be reached by the request.
+        :param dict payload: The payload of the request.
+        :return The JSON-formatted content of the response.
+        :rtype: dict
+        """
+        endpoint = f'{self._get_pinelabs_endpoint()}{endpoint}'
+        try:
+            response = self.session.post(endpoint, json=payload, timeout=REQUEST_TIMEOUT)
```

Comment:
why not ?
```suggestion
            response = requests.post(endpoint, json=payload, timeout=REQUEST_TIMEOUT)
```

Reply:
I have done according to [this](https://github.com/odoo/enterprise/wiki/security#PY006).

What is your thought?

Reply:
![image](https://github.com/user-attachments/assets/84289a57-81ae-4aa6-8810-5828eb034b73)